cognite/replicator/replication.py

```diff
@@ -194,6 +202,9 @@ def make_objects_batch(
                 update_objects.append(dst_obj)
             else:
                 unchanged_objects.append(dst_obj)
+        elif dst_ts:
+            if contains(dst_ts, lambda x: x.external_id == "VAL_23-PT-9253:X.Value"):
```

Comment:
See comment above, with any. Also there is a hardcoded matching here against "VAL_23-PT-9253:X.Value".

Reply:
Thanks, fixed